test/endtoend/operator/101_initial_cluster_vtadmin.yaml

```diff
@@ -0,0 +1,231 @@
+# The following example is minimalist. The security policies
+# and resource specifications are not meant to be used in production.
+# Please refer to the operator documentation for recommendations on
+# production settings.
+apiVersion: planetscale.com/v2
+kind: VitessCluster
+metadata:
+  name: example
+spec:
+  images:
+    vtctld: vitess/lite:v14.0.0-rc1
+    vtadmin: vitess/vtadmin:latest
```

Comment:
Yeah, I think `latest` is fine here. We will change it tomorrow during the release

Reply:
Yes, exactly